### PR TITLE
[HyperShift/KubeVirt] Add e2e test for external infra

### PR DESCRIPTION
**Pull request by orenc1**
This PR adds another run of the e2e test suite, but in the second time it uses external infra logic, by providing infra cluster kubeconfig and infra cluster namespace. The external cluster is actually the mgmt cluster, using its kubeconfig, but the logic that will be tested is the external infra one.

### CODE REVIEW

File: ci-operator/config/openshift/hypershift/openshift-hypershift-main__kubevirt.yaml
Comment (davidvossel) [inline ci-operator/config/openshift/hypershift/openshift-hypershift-main__kubevirt.yaml:119]: fyi, I don't think you have to run these commands again. It doesn't hurt anything, it's just a no-op

**Review by davidvossel — COMMENTED** [review]
/approve
/lgtm
/hold

looks great, the hold is for us to get a passing CI in place before this is merged

File: ci-operator/config/openshift/hypershift/openshift-hypershift-main__kubevirt.yaml
Comment (orenc1) [reply ci-operator/config/openshift/hypershift/openshift-hypershift-main__kubevirt.yaml]: I agree.
Removed redundant commands.

**Comment by orenc1** [comment]
/pj-rehearse pull-ci-openshift-hypershift-main-e2e-kubevirt-aws-ovn
/pj-rehearse pull-ci-openshift-hypershift-release-4.13-e2e-kubevirt-aws-ovn

**Comment by orenc1** [comment]
https://github.com/openshift/hypershift/pull/2439 merged, can test 4.13
/pj-rehearse pull-ci-openshift-hypershift-release-4.13-e2e-kubevirt-aws-ovn

**Comment by davidvossel** [comment]
/pj-rehearse pull-ci-openshift-hypershift-release-4.13-e2e-kubevirt-aws-ovn

**Comment by orenc1** [comment]
> /pj-rehearse pull-ci-openshift-hypershift-release-4.13-e2e-kubevirt-aws-ovn

it passed in previous run, you want to check for consistency?
btw, the conditional worked and external flow was not executed:
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/37755/rehearse-37755-pull-ci-openshift-hypershift-release-4.13-e2e-kubevirt-aws-ovn/1648973527467954176/artifacts/e2e-kubevirt-aws-ovn/run-e2e-external/build-log.txt

**Comment by davidvossel** [comment]
/pj-rehearse pull-ci-openshift-hypershift-main-e2e-kubevirt-aws-ovn

**Comment by davidvossel** [comment]
/pj-rehearse pull-ci-openshift-hypershift-main-e2e-kubevirt-aws-ovn

**Comment by davidvossel** [comment]
@orenc1 I think we might have to get this merged (https://github.com/openshift/hypershift/pull/2466) before the main branch rehearsal will pass

**Comment by orenc1** [comment]
https://github.com/openshift/hypershift/pull/2466 merged.
/pj-rehearse pull-ci-openshift-hypershift-main-e2e-kubevirt-aws-ovn

**Comment by orenc1** [comment]
/pj-rehearse pull-ci-openshift-hypershift-main-e2e-kubevirt-aws-ovn

**Comment by orenc1** [comment]
/pj-rehearse pull-ci-openshift-hypershift-main-e2e-kubevirt-aws-ovn

**Comment by orenc1** [comment]
/pj-rehearse pull-ci-openshift-hypershift-main-e2e-kubevirt-aws-ovn

**Comment by orenc1** [comment]
/pj-rehearse pull-ci-openshift-hypershift-main-e2e-kubevirt-aws-ovn

**Comment by orenc1** [comment]
/pj-rehearse pull-ci-openshift-hypershift-main-e2e-kubevirt-aws-ovn

**Review by davidvossel — APPROVED** [review]
/lgtm
/approve

Yay!

**Pull request merged**
